src/Mesh/Grids.jl

```diff
@@ -278,14 +279,19 @@ function computegeometry(topology::AbstractTopology{dim}, D, ξ, ω, meshwarp,
     Metrics.computemetric!(x1, x2, J, ξ1x1, ξ2x1, ξ1x2, ξ2x2, sJ, n1, n2, D)
   elseif dim == 3
     Metrics.computemetric!(x1, x2, x3, J, ξ1x1, ξ2x1, ξ3x1, ξ1x2, ξ2x2, ξ3x2,
-                           ξ1x3, ξ2x3, ξ3x3, sJ, n1, n2, n3, D)
+                           ξ1x3, ξ2x3, ξ3x3, sJ, n1, n2, n3, D, JH)
   end
 
   M = kron(1, ntuple(j->ω, dim)...)
   MJ .= M .* J
   MJI .= 1 ./ MJ
   vMJI .= MJI[vmapM]
 
+  MH = kron(1, ntuple(j->ω, dim)...)
```

Comment:
This quantity is a surface quantity that lives in the element volume. Is this why you use `dim` and not `dim-1` in the Kronecker function? @slifer50 @jkozdon